entry_types/scrolled/package/src/frontend/VideoPlayer.js

```diff
@@ -0,0 +1,72 @@
+import React, {useRef, useEffect, useContext} from 'react';
+import classNames from 'classnames';
+
+import ScrollToSectionContext from './ScrollToSectionContext';
+import MutedContext from './MutedContext';
+
+import styles from './VideoPlayer.module.css';
+
+/**
+ * Render a player for a given video file.
+ *
+ * @param {Object} props
+ * @param {number} props.videoFile - The file to be played
+ */
+export function VideoPlayer(props) {
```

Comment:
This component is an implementation detail of the `Video` component. I would have probably left it as a private function inside the same file, but that might be a matter of taste.

Adding inline docs, though, makes it appear in the API doc even though it is not exported in `frontend/index.js` and can thus not be used outside of the `frontend` module.